### DISCUSSION TITLE
2024-09-09-future

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#189] [General]
# 聊聊未来技术趋势 - Tw93

Hi, I'm Tw93, a Frontend Engineer from China, also dabbling a bit in Swift, Rust, Python, and Sketch. I'm the creator of MiaoYan, Pake, Trendy Weekly, XRender, and WeexUi. Believing in long-termism, I am passionate about open source and enjoy experimenting with new ideas.Share最近在团队的一个分享，基于自己对 AI、投资、下一代技术的一些理解，结合网络上的报告、资料、论文、财报等简单梳理了下，假如有不懂的名词可以问 GPT，希望你开心，但不要在不懂市场的情况下基于此去投资，风险会很高。

https://tw93.fun/2024-09-09/future.html

**Comment by print248** [comment]
大佬，请问您PPT和里面的图表都是怎么制作的，感觉好看又专业

**Reply by tw93** [reply]
keynote 就够了， 不过这里面有不少我是从英文报告里面找到的，然后我照着给整理了下。

**Comment by josangmi** [comment]
突然想要个原文件。

**Reply by bbingz** [reply]
我也是，能分享么？

**Reply by tw93** [reply]
已经补充，可以在原文底部获取到 pdf 文件，点击就可以下载。

**Comment by jxderic** [comment]
大佬，有分享的录屏吗？

**Reply by tw93** [reply]
抱歉 没有，不过有啥不清楚的，可以问我

**Comment by miniLCT** [comment]
作者是不是炒币😊
